rmgpy/rmg/input.py

```diff
@@ -104,6 +104,9 @@ def database(
 
 def species(label, structure, reactive=True):
     logging.debug('Found {0} species "{1}" ({2})'.format('reactive' if reactive else 'nonreactive', label, structure.toSMILES()))
+    
+    assert not '+' in label, 'species labels cannot include a + sign'
```

Comment:
I think this could actually be replaced by raising an `InputError` to make it clearer to the user. I think assertions should generally be for things that should never happen.

Reply:
Ok, done